node/tessel-export.js

```diff
@@ -952,6 +957,303 @@ Tessel.LED.prototype.read = function(callback) {
   });
 };
 
+Tessel.Wifi = function() {
+  var state = {
+    settings: {},
+    connected: false
+  };
+
+  Object.defineProperties(this, {
+    isConnected: {
+      get: () => state.connected
+    },
+    connected: {
+      set: (value) => {
+        state.connected = value;
+      }
+    },
+    settings: {
+      get: () => state.settings,
```

Comment:
How could we keep that `state` updated without exposing it to unprivileged access? Do we `bind` the `state` object to each method?